drivers/SmartThings/zigbee-thermostat/src/popp_danfoss/init.lua

```diff
@@ -0,0 +1,69 @@
+local clusters = require "st.zigbee.zcl.clusters"
+local capabilities = require "st.capabilities"
+local utils = require "st.utils"
+local Thermostat = clusters.Thermostat
+local PowerConfiguration = clusters.PowerConfiguration
+local PowerSource = capabilities.powerSource
+local Battery = capabilities.battery
+local ThermostatMode = capabilities.thermostatMode
+
+local BAT_MIN = 24.0
+local BAT_MAX = 32.0
+
+local POPP_DANFOSS_THERMOSTAT_FINGERPRINTS = {
+  { mfr = "D5X84YU", model = "eT093WRO" },
+  { mfr = "Danfoss", model = "eTRV0100" }
+}
+
+local is_popp_danfoss_thermostat = function(opts, driver, device)
+  for _, fingerprint in ipairs(POPP_DANFOSS_THERMOSTAT_FINGERPRINTS) do
+    if device:get_manufacturer() == fingerprint.mfr and device:get_model() == fingerprint.model then
+      return true
+    end
+  end
+  return false
+end
+
+local thermostat_heating_setpoint_handler = function(driver, device, value)
+  device:emit_event(capabilities.thermostatHeatingSetpoint.heatingSetpoint({value = value.value/100.0, unit = "C"}))
+end
```

Comment:
Should be removed since it is handled in the same way as defaults

Reply:
> Should be removed since it is handled in the same way as defaults

ok